libs/ui/ui-shared/src/lib/components/Sort/Sort.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Select } from 'antd';
+import styles from './Sort.module.scss';
+
+export const Sort = () => {
+  const handleChange = (value: string) => {
+    console.log(`selected ${value}`);
+  };
+
+  return (
+    <div className={styles.sort}>
```

Comment:
Do we really need to add flex and gap to this div?
And can we remove this div wrapper?